### DISCUSSION TITLE
Best way to modify existing models?

### DISCUSSION THREAD

**Discussion by a3rosol** [loco-rs/loco#635] [Q&A] [answered]
I'm completely new to loco and was wondering how I can add or remove fields from existing models. Say I'd like to add "language_id" to users. The docs describe how to create new models but not necessarily how to modify them. This includes deletion: how can I safely delete "notes" for example?

Is there a cli command to do this or do I have to manually edit migrations?

**Comment by kaplanelad** [comment, marked as answer]
Hey @a3rosol 

You have two options:
1. In case you didn't run the migration on the production, you can modify the existing migration script, delete the table from your local DB and the migration DB row and rerun the migration
2. If it's on production, you can create another migration which modifies the table and drops/adds the column


You can follow seaorm guide here: 
https://www.sea-ql.org/SeaORM/docs/next/migration/writing-migration/
